css/style.css

```diff
@@ -9,400 +14,622 @@ body {
-        margin-top: 5%;
+@media (max-width: 1200px) { 
+  .girl {
+    width: 45%;
+    right: 60%;
     }
 }
-    
-@media (max-width: 1270px) { 
-    .girl {
-        width: 50%;
-        margin-left: 5%;
+ 
+@media (max-width: 1200px) { 
+  .new-сollection__main-button {
+    width: 35%;
+    font-size: 8px;
+     }
+}
+ 
+@media (max-width: 1200px) { 
+  .new-сollection__main-link {
+    width: 50%;
+    margin-left: 3%;
+    font-size: 10px;
+    margin-right: -15%;
+     }
+}
+ 
+@media (max-width: 1200px) { 
+  .new-сollection-arrow {
+    width: 10%;
+     } 
+}
+
+@media (max-width: 1200px) {
+  .new-сollection__social {
+    width: 47%;
+     }
+}
+
+@media (max-width: 961px) { 
+  .new-сollection__main-card {
+    margin-top: 5%;
     }
 }
 
-@media (max-width: 1100px) { 
-    .new-сollection__main-button {
-        width: 35%;
+@media (max-width: 961px) { 
+  .new-сollection__main-link {
+    margin-right: -5%;
     }
 }
 
-@media (max-width: 1100px) { 
-    .new-сollection__main-link {
-        width: 50%;
-        margin-left: 3%;
+@media (max-width: 961px) { 
+   .new-collection__background {
+    max-width: 45%;
+    min-height: 900px;
     }
 }
 
-@media (max-width: 1100px) { 
-     .new-сollection-arrow {
-        width: 10%;
-    } 
-     }
+@media (max-width: 961px) { 
+  .logo {
+    margin-right: 20%;
+    }
+}
 
+@media (max-width: 961px) {
+  .new-сollection-main-link {
+    margin-left: 5%;
+   } 
+}
 
 @media (min-width: 800px) { 
-    .logo-mobil,
-    .galka,
-    .new-collection__background_girl {
-        display: none;
-        } 
+  .logo-mobil,
+  .galka,
+  .new-collection__background_girl {
+    display: none;
     } 
-
+} 
+ 
 @media (max-width: 799px) { 
-    .logo,
-    .lang,
-    .check,
-    .new-сollection__social,
-    .text,
-    .new-сollection__main-card,
-    .new-collection__background {
-         display: none;
-        } 
+  .logo,
+  .lang,
+  .check,
+  .new-сollection__social,
+  .text,
+  .new-сollection__main-card,
+  .new-collection__background {
+    display: none;
     } 
-
+} 
+ 
 @media (max-width: 800px) { 
-    .logo-mobil {
-          position: absolute;
-          width: 98px;
-          height: 37px;
-          padding: 5% 0 0 7%; 
-        }
-    }  
-
-@media (max-width: 799px) { 
-    .girl {
-        background: none
-     }
+  .logo-mobil {
+    position: absolute;
+    width: 98px;
+    height: 37px;
+    padding: 5% 0 0 7%;
     }
-
+}  
+ 
 @media (max-width: 799px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-size: cover;
-        background-position: 0 92px; 
-        width: 80%;
-        margin-left: 10%;
-        position: absolute;
-        
-    }
- }
-
- @media (max-width: 799px) { 
-    .title {
-        top: 80%;
-        width: 15%;
-        font-size: 50px;
-        line-height: 52px;
-        left: 30%;
+  .girl {
+    background: none;
     }
-     } 
-
+}
+ 
+@media (max-width: 799px) { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)